spec/models/beach_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+RSpec.describe Beach, type: :model do
+  let(:beach) { FactoryGirl.create(:beach) }
+
+  describe "validations" do
+    it { should validate_presence_of(:street1) }
+    it { should validate_presence_of(:city) }
+    it { should validate_presence_of(:state) }
+    it { should validate_presence_of(:zip) }
+    subject { Beach.new(
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.